datasets/aev-a2d2.yaml

```diff
@@ -0,0 +1,31 @@
+Name: A2D2: AEV Autonomous Driving Dataset
+Description: |
+        An open multi-sensor dataset for autonomous driving research.
+        This dataset comprises semantically segmented images, semantic
+        point clouds, and 3D bounding boxes. In addition, it contains
+        unlabelled 360 degree camera images, lidar, and bus data for
+        three sequences. We hope this dataset will further facilitate
+        active research and development in AI, computer vision, and
+        robotics for autonomous driving. Commercial use is permitted
+        as per the CC BY-ND 4.0 license.
+Contact: aevdrivingdataset@audi.de
+Documentation: https://www.audi-electronics-venture.de/driving-dataset.html
+ManagedBy: Audi Electronics Ventures GmbH, 85080 Gaimersheim, Germany
```

Comment:
Can you change this to something like "[Audi Electronics Ventures](https://www.audi-electronics-venture.de)"? No need for the actual address here.